src/shared/base.props

```diff
@@ -0,0 +1,27 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
If we need not share config among csprojs, we can embed this directly into csproj, and abandon this file.